src/Symfony/Component/EventDispatcher/Tests/DependencyInjection/RegisterListenersPassTest.php

```diff
@@ -503,6 +507,27 @@ public function testOmitEventNameOnSubscriber()
         ];
         $this->assertEquals($expectedCalls, $definition->getMethodCalls());
     }
+
+    /**
+     * @group legacy
+     */
+    public function testEventDeprecations()
+    {
+        $this->expectDeprecation('Since symfony/event-dispatcher 7.1: Event is deprecated');
```

Comment:
Spult be moved before the compile() method, as this triggers the deprecation, does it?

Reply:
Indeed, done